src/screens/Survey/lib/questions.json

```diff
@@ -0,0 +1,55 @@
+{
+        "question1":
+            {
+            "id":"1",
+            "qtext":"Should the government guarantee healthcare for all Americans?",
```

Comment:
Is this mock data for development? These keys don't match what is coming back from the spreadsheet.  I think you should either update the keys to match the data being used, or delete the file if it is no longer being used.

Reply:
Oh right, I should remove that actually. Sorry my bad